### DISCUSSION TITLE
My account suspended

### DISCUSSION THREAD

**Discussion by AnLaVN03** [community/community#141003] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Account Name: AnLaVN https://github.com/AnLaVN

My account has been suspended for 3 months, I have important projects in my repository. I really need to access my account again. How can we solve this serious problem?

I was send 2 ticket 2942666, 2956836 but my account still suspended, please help.

**Comment by xastrix** [comment]
1. Go to the GitHub Support page: [Github Support](https://support.github.com/contact)
2. Select the appropriate category: Find the section related to account issues.
3. Describe your situation: Provide all relevant details about your account and the reason for the lockout.
4. Submit your request: Fill out the form and submit it for review.

**Reply by xastrix** [reply]
If your account is still locked, it may not be unlocked ever again, I'm sorry.

**Reply by AnLaVN03** [reply]
i filled out a support ticket and they wont let me create another one because of duplicate tickets, they didnt respond at all, my account is gone and all my work has been seriously affected, i really need my account back. the last time i filled out a support ticket was yesterday and they needed sms verification but i am in vietnam now and they dont support my country's phone number. i really need my account :((((

**Comment by ebndev** [comment]
Hi @AnLaVN03,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
